rules-reviewed/eap6/java-ee/java-ee-jaxrpc.windup.xml

```diff
@@ -22,14 +22,22 @@
                     <location>INHERITANCE</location>
                 </javaclass>
             </when>
-            <perform>
-                <classification title="JAX-RPC Generic Handler" effort="1" severity="mandatory">
-                    <link href="http://docs.oracle.com/cd/E17802_01/webservices/webservices/docs/1.6/api/javax/xml/rpc/handler/GenericHandler.html"
-                        title="Java EE RPC Generic Handler API" />
-                    <tag>jax-rpc</tag>
-                    <tag>soap</tag>
-                </classification>
-            </perform>
-        </rule>
+              <perform>
+                  <hint title="JAX-RPC GenericHandler usage" effort="0" severity="optional">
```

Comment:
not really sure why this should be a hint as the hint message doesn't recommend anything useful how to migrate.
I think here is the correctly used *classification* type and we probably use message as a classification description.

Btw, @jsight/@OndraZizka why don't we format classification description to markdown format?